spotbugs-tests/src/test/java/edu/umd/cs/findbugs/nullness/JakartaNullabilityTest.java

```diff
@@ -0,0 +1,43 @@
+package edu.umd.cs.findbugs.nullness;
+
+import edu.umd.cs.findbugs.BugCollection;
+import edu.umd.cs.findbugs.test.SpotBugsExtension;
+import edu.umd.cs.findbugs.test.SpotBugsRunner;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.nio.file.Paths;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.emptyIterable;
+
+/**
+ * Check if Nullable annotation from org.apache.avro.reflect is detected.
+ *
+ * @author pkini07
+ */
+@ExtendWith(SpotBugsExtension.class)
+class JakartaNullabilityTest {
+
+    @Test
+    void checkedJakartaNullableReturn_isOk(SpotBugsRunner spotbugs) {
+        BugCollection bugCollection = spotbugs.performAnalysis(
+                Paths.get("../spotbugsTestCases/build/classes/java/main/CheckedJakartaNullableReturn.class"));
+        assertThat(bugCollection, emptyIterable());
```

Comment:
Can you please assert that, there is no bug of one or several specific bugtypes? So the test won't break if a new bug would be introduced, which finds an unrelated bug in the example.